.vscode/settings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "javascript.format.enable": false,
+  "typescript.format.enable": false,
+  "json.format.enable": false,
+
+  "js/ts.implicitProjectConfig.checkJs": false,
+  "css.validate": false,
+
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "[python]": { "editor.formatOnSave": false },
```

Comment:
Note that I disable auto-formatting for Python, at least for now.

Reply:
Why so ?

Reply:
Okay, so turns out it was just a matter of configuring the Black extension:

```jsonc
"black-formatter.importStrategy": "fromEnvironment" // "fromBundle" by default
```